### ISSUE TITLE
Cannot use bower with more recent version of Chart.js

### ISSUE THREAD

**Issue by rjfranco** [Regaddi/Chart.StackedBar.js#20]
When packaging manually, seems to work, but would like to be able to upgrade through the bower package manager.

**Comment by amiuhle** [comment]
Probably related to nnnick/Chart.js#915

Use `Chart.js` in your bower.json and specify version 1.0.1. If you run `bower update`, it will ask you to specify a resolution.

**Comment by rjfranco** [comment]
I don't actually use bower locally, I use rails-assets.org which takes bower components and makes them available as gems in rails projects, I can manually include the library of course and it's been great, thank you!

**Comment by amiuhle** [comment]
Yeah, rails-assets.org is a nice service. Try using `Chart.js` as the bower package name instead of `chartjs`.

**Comment by rjfranco** [comment]
Actually I was thinking it was related to my pull request: https://github.com/Regaddi/Chart.StackedBar.js/pull/21 but I think the project might need a version bump for rails-assets to check for a change ( unless there's a way to manually trigger that I couldn't find ), the error produced is:

```
Bundler could not find compatible versions for gem "rails-assets-Chart.js":
  In Gemfile:
    rails-assets-Chart.StackedBar.js (~> 1.0.1) ruby depends on
      rails-assets-Chart.js (= 1.0.0.beta) ruby

    rails-assets-Chart.js (1.0.1)
```

**Comment by Regaddi** [comment]
Still a problem?

**Comment by rjfranco** [comment]
This still seems to be an issue :-/
